In [1]:
expenses = [1200, 1500, 800, 600, 1000, 400, 900]

total = sum(expenses)
average = total / len(expenses)
maximum = max(expenses)
minimum = min(expenses)

days_above_1000 = sum(1 for x in expenses if x > 1000)

print(total, average, maximum, minimum, days_above_1000)

6400 914.2857142857143 1500 400 2


| Показник | Значення |
|-----------|-----------|
| Загальна сума витрат | 6400 |
| Середнє значення | 914.29 |
| Максимальна витрата | 1500 |
| Мінімальна витрата | 400 |
| Днів з витратами понад 1000 | 2 |